docs/core/idioms/pause-for-approval.md

```diff
@@ -0,0 +1,53 @@
+# Pause for Approval
```

Comment:
Could you add this to the list on `idioms.md`?

Reply:
Added.